api/src/config.rs

```diff
@@ -820,7 +820,7 @@ pub struct RafsConfigV2 {
     #[serde(default = "default_rafs_mode")]
     pub mode: String,
     /// Batch size to read data from storage cache layer.
-    #[serde(default = "default_batch_size")]
+    #[serde(default = "default_user_io_batch_size")]
```

Comment:
Compatibility is required.

Reply:
This setting is used to set the default value function, and does not change the variable names in the config file. So there's no compatibility issue.